solverdummy/solverdummy.jl

```diff
@@ -1,10 +1,11 @@
-import Pkg; Pkg.activate("..")
+# import Pkg; Pkg.activate("..")
```

Comment:
Can we just remove this line if the solverdummy test script works?

Reply:
I left it in, in case something would break at a later point.
To my understanding, [`Pkg.activate("..")`](https://docs.julialang.org/en/v1/stdlib/Pkg/) does something similar to python virtual-environments. I don't see the use of activating a specific environment at the start of the dummy. Without the activate the dummy works in both cases, with the test and when started manually. 
So I think it is safe to remove this line for now.